src/stilling/stilling/forhåndsvisning/Forhåndsvisning.tsx

```diff
@@ -13,14 +13,19 @@ import Søknad from './søknad/Søknad';
 
 type Props = {
     stilling: Stilling;
+    erFormidling: boolean;
 };
 
-const Forhåndsvisning = ({ stilling }: Props) => {
+const Forhåndsvisning = ({ stilling, erFormidling }: Props) => {
     return (
         <div className={css.forhåndsvisning}>
             <div className={css.venstre}>
                 <Panel as="article" className={css.annonsetekst}>
-                    {parse(stilling.properties.adtext || '')}
+                    {erFormidling ? (
+                        <span>Formidling</span>
+                    ) : (
+                        parse(stilling.properties.adtext || '')
```

Comment:
Burde dette vært fjernet i backend? Er det greit at teksten kan ses i devconsole for gamle stillinger som ikke er redigert?

Reply:
Gjerne luft det med Per Erik igjen, men tok denne i frontend nå da det var lavthengende frukt